### PR TITLE
🐛 bug: Use Content-Length for bytesReceived and bytesSent tags in Logger Middleware

### LINKED ISSUE #3060: 🐛 [Bug]: Logger module causes OOM with ${bytesSent} and ${bytesReceived}
### Bug Description

When download large files the process can be OOM'd. I have tried serving ~2GB ISOs and I've noticed that memory is not reclaimed.

### How to Reproduce

1. Setup a go-fiber app using static being sure to add a large file (such as an ISO) to the static directory, for example:

```go
	app.Static("/", "./mydir", fiber.Static{
		Download: true,
	})
```

2. Download with curl multiple times (until memory is exhausted): `curl localhost/isos/some.iso`
3. This will result in a large amount of memory being used. Under normal circumstances about `33MB` RSS is used. Once I've downloaded a 2GB iso, the process is now using `4128M`. If I download the file again, this amount of memory increases. Eventually the process is OOM'd.
4. I have tried tweaking some functionality such as disabling keep-alives as well as setting `Download: true` (in static) and `CacheDuration: -1`

pprof (if it helps):

(-inuse_space)
```
Showing nodes accounting for 2GB, 99.92% of 2GB total
Dropped 8 nodes (cum <= 0.01GB)
Showing top 10 nodes out of 11
      flat  flat%   sum%        cum   cum%
       2GB 99.92% 99.92%        2GB 99.92%  github.com/valyala/bytebufferpool.(*ByteBuffer).ReadFrom
         0     0% 99.92%        2GB 99.92%  github.com/gofiber/fiber/v2.(*App).handler
         0     0% 99.92%        2GB 99.92%  github.com/gofiber/fiber/v2.(*App).next
         0     0% 99.92%        2GB 99.92%  github.com/gofiber/fiber/v2/middleware/logger.New.func3
         0     0% 99.92%        2GB 99.92%  github.com/gofiber/fiber/v2/middleware/logger.createTagMap.func12
         0     0% 99.92%        2GB 99.92%  github.com/valyala/fasthttp.(*Response).Body
         0     0% 99.92%        2GB 99.92%  github.com/valyala/fasthttp.(*Server).serveConn
         0     0% 99.92%        2GB 99.92%  github.com/valyala/fasthttp.(*bigFileReader).WriteTo
         0     0% 99.92%        2GB 99.92%  github.com/valyala/fasthttp.(*workerPool).getCh.func1
         0     0% 99.92%        2GB 99.92%  github.com/valyala/fasthttp.(*workerPool).workerFunc
```

(-alloc_space)

```
Showing top 10 nodes out of 11
      flat  flat%   sum%        cum   cum%
      12GB   100%   100%       12GB   100%  github.com/valyala/bytebufferpool.(*ByteBuffer).ReadFrom
         0     0%   100%       12GB   100%  github.com/gofiber/fiber/v2.(*App).handler
         0     0%   100%       12GB   100%  github.com/gofiber/fiber/v2.(*App).next
```

### Expected Behavior

File should be served without OOMing.

### Fiber Version

v2.52.4

### Code Snippet (optional)

```go
package main

import (
	"log"
	"os"

	"github.com/gofiber/fiber/v2"
	"github.com/gofiber/fiber/v2/middleware/logger"
)

func main() {
	app := fiber.New()

	// comment out to "fix" bug
	app.Use(logger.New(logger.Config{
		TimeZone:   "UTC",
		TimeFormat: "02/Jan/2006:15:04:05 -0700",
		Format:     "${ip} - - [${time}] \"${method} ${path}${queryParams} HTTP/1.1\" ${status} ${bytesSent} \"${referer}\" \"${ua}\"\n",
		Output:     os.Stdout,
	}))

	app.Static("/", "./static", fiber.Static{
		Browse:        true,
		CacheDuration: -1,
	})

	v1 := app.Group("/api/v1")

	v1.Get("/start", func(c *fiber.Ctx) error {
		return nil
	})
	v1.Get("/start2", func(c *fiber.Ctx) error {
		return nil
	})

	v1.Get("/start3", func(c *fiber.Ctx) error {
		return nil
	})
	v1.Get("/start4", func(c *fiber.Ctx) error {
		return nil
	})

	log.Fatal(app.Listen(":3000"))
}
```

### Checklist:

- [X] I agree to follow Fiber's [Code of Conduct](https://github.com/gofiber/fiber/blob/master/.github/CODE_OF_CONDUCT.md).
- [X] I have checked for existing issues that describe my problem prior to opening this one.
- [X] I understand that improperly formatted bug reports may be closed without explanation.

### PR DESCRIPTION
**Pull request by gaby**
## Description

- Use `Content-Length` header when calculating the `bytesSent` and `bytesReceived` tags in the Logger Middleware.
  - The current implementation will load big files to memory to calculate the len causing Fiber to use a high amount of RAM.
- Added new test case: `Test_Request_Body` for testing the `bytesReceived` tag.
- Added test case using `SetContentLength`

Fixes #3060 

## Type of change

- [x] Performance improvement (non-breaking change which improves efficiency)
- [x] Code consistency (non-breaking change which improves code reliability and robustness)

### CODE REVIEW

File: middleware/logger/tags.go [hunk 1]
```diff
@@ -87,13 +88,10 @@ func createTagMap(cfg *Config) map[string]LogFunc {
 			return output.Write(c.Body())
 		},
 		TagBytesReceived: func(output Buffer, c fiber.Ctx, _ *Data, _ string) (int, error) {
-			return appendInt(output, len(c.Request().Body()))
+			return output.WriteString(strconv.Itoa((c.Request().Header.ContentLength())))
```
Comment (ReneWerner87) [inline middleware/logger/tags.go:91]: strconv.Itoa is very slow can we find a better solution ?

Comment (gaby) [reply middleware/logger/tags.go:91]: Will be fixing this tonight.

Comment (ReneWerner87) [reply middleware/logger/tags.go:91]: @gaby use https://github.com/gofiber/utils/pull/90 ater merge

Comment (ReneWerner87) [reply middleware/logger/tags.go:91]: @gaby 
i closed https://github.com/gofiber/utils/pull/90#issuecomment-2242201778

pls use https://pkg.go.dev/strconv#AppendInt

Comment (gaby) [reply middleware/logger/tags.go]: Done

**Review by ReneWerner87 — APPROVED** [review]

**Pull request merged**